lib/tests/streamlit/web/server/server_test.py

```diff
@@ -13,66 +13,60 @@
 # limitations under the License.
 
 """Server.py unit tests"""
+
 import asyncio
 import errno
 import os
 import shutil
 import tempfile
 import unittest
 from unittest import mock
-from unittest.mock import MagicMock, patch
+from unittest.mock import patch
 
 import pytest
 import tornado.httpserver
+import tornado.httpserver
+import tornado.httpserver
 import tornado.testing
+import tornado.testing
+import tornado.testing
+import tornado.web
 import tornado.web
+import tornado.web
+import tornado.websocket
+import tornado.websocket
 import tornado.websocket
```

Comment:
Hm, it looks like a bunch of these imports got repeated somehow

Reply:
haha, thanks PyCharm!